### DISCUSSION TITLE
Collapse with the seperator

### DISCUSSION THREAD

**Discussion by vikdb** [Clooos/Bubble-Card#945] [Questions about config, custom styles and templates]
I've see a few examples of people using the collapse icon in a separator to hide part of the UI.
Do you guys use toggle switches for all these in combination with a visibility condition? or is there some CSS magic that is used? 

As creating all these toggles is quite the job and syncs over all users.

**Comment by SylvainDx** [comment]
do you have a screen shot of your topic ?

**Comment by MrBearPresident** [comment]
Did you solve this?
Can this be closed?
